clients/venice-admin-tool/src/main/java/com/linkedin/venice/KafkaTopicDumper.java

```diff
@@ -42,35 +52,42 @@
 import org.apache.avro.io.DatumWriter;
 import org.apache.avro.io.Decoder;
 import org.apache.avro.io.DecoderFactory;
+import org.apache.avro.specific.SpecificDatumReader;
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
 
 
 public class KafkaTopicDumper implements AutoCloseable {
   private static final Logger LOGGER = LogManager.getLogger(KafkaTopicDumper.class);
-  public static final String VENICE_ETL_KEY_FIELD = "key";
-  public static final String VENICE_ETL_VALUE_FIELD = "value";
-  public static final String VENICE_ETL_OFFSET_FIELD = "offset";
-  public static final String VENICE_ETL_DELETED_TS_FIELD = "DELETED_TS";
-  public static final String VENICE_ETL_METADATA_FIELD = "metadata";
+  private static final String VENICE_ETL_KEY_FIELD = "key";
+  private static final String VENICE_ETL_VALUE_FIELD = "value";
+  private static final String VENICE_ETL_OFFSET_FIELD = "offset";
+  private static final String VENICE_ETL_DELETED_TS_FIELD = "DELETED_TS";
+  private static final String VENICE_ETL_METADATA_FIELD = "metadata";
 
-  public static final String VENICE_ETL_BROKER_TIMESTAMP_FIELD = "brokerTimestamp";
-  public static final String VENICE_ETL_PRODUCER_TIMESTAMP_FIELD = "producerTimestamp";
-  public static final String VENICE_ETL_PARTITION_FIELD = "partition";
+  private static final String VENICE_ETL_BROKER_TIMESTAMP_FIELD = "brokerTimestamp";
+  private static final String VENICE_ETL_PRODUCER_TIMESTAMP_FIELD = "producerTimestamp";
+  private static final String VENICE_ETL_PARTITION_FIELD = "partition";
 
   private final String storeName;
   private final String topicName;
   private final int partition;
   private final String keySchemaStr;
   private final String latestValueSchemaStr;
   private final Schema[] allValueSchemas;
+  private final boolean isChunkingEnabled;
   private final String parentDirectory;
   private final PubSubConsumerAdapter consumer;
   private final long messageCount;
   private final long endOffset;
   private final int maxConsumeAttempts;
   private final boolean logMetadataOnly;
 
+  private final ChunkKeyValueTransformer chunkKeyValueTransformer;
+  private final SpecificDatumReader<ChunkedKeySuffix> specificDatumReader;
```

Comment:
This one can be a local variable instead of a class-level and also the naming is confusing when sitting in class level.

Reply:
Yeah, I agree